layers/timeline_semaphore.c

```diff
@@ -1337,6 +1338,7 @@ vk_submit_structure_type_size(const VkBaseInStructure *item)
     default:
         unreachable("Unknown structure for VkSubmitInfo::pNext");
     }
+    return 0;
```

Comment:
Resolves the following warning in a release build:

../third_party/vulkan-extensionlayer/layers/timeline_semaphore.c: In function ‘vk_submit_structure_type_size’:
../third_party/vulkan-extensionlayer/layers/timeline_semaphore.c:1341:1: warning: control reaches end of non-void function [-Wreturn-type]